workshops/dodge/README.md

```diff
@@ -607,31 +607,31 @@ When we add an image to a sprite, the sprite's width and height is changed to ma
 
 ```js
 // in the setup function...
-̶p̶l̶a̶y̶e̶r̶ ̶=̶ ̶c̶r̶e̶a̶t̶e̶S̶p̶r̶i̶t̶e̶(̶w̶i̶d̶t̶h̶/̶2̶,̶ ̶h̶e̶i̶g̶h̶t̶-̶2̶5̶,̶ ̶5̶0̶,̶ ̶5̶0̶)̶;̶
+~~player = createSprite(width/2, height -25, -50, 50),~~
```

Comment:
Looks like they support git diffs (https://highlightjs.org/static/demo/#diff). Thoughts on using that?